egs/callhome_diarization/v2/run.sh

```diff
@@ -92,7 +109,29 @@ if [ $stage -le 1 ]; then
   # detection (SAD) system, but this is not necessary.  You can replace
   # this with segments computed from your favorite SAD.
   diarization/vad_to_segments.sh --nj 40 --cmd "$train_cmd" \
-    data/sre_cmn data/sre_cmn_segmented
+				 data/sre_cmn data/sre_cmn_segmented
+
+  if [ $VB_resegmentation ]; then
+      # Prepare telephone and microphone speech from Mixer6.
+      local/make_mx6.sh $data_root/LDC2013S03 data/
+
+      # Prepare SRE10 test and enroll. Includes microphone interview speech.
+      # NOTE: This corpus is now available through the LDC as LDC2017S06.
+      local/make_sre10.pl $sre_root/SRE2010/eval/ data/
```

Comment:
I don't understand why the VB resegmentation requires a separate set of datasets than the rest of the system. Why can't we train it on the same datasets used to train the x-vector DNN?

Unless there's a good reason, we should just use the initial datasets.

Reply:
We used these recent versions of data - sre10 and mx6 in order to obtain the claimed results.